### ISSUE TITLE
Font issue on Big Sur

### ISSUE THREAD

**Issue by tomtastic** [Col-E/Recaf#238]
**Describe the bug**

Font garbled on MacOS Big Sur (probably MacOS Catalina too).

**To Reproduce**

```
$ java --version
openjdk 11.0.8 2020-07-14
OpenJDK Runtime Environment (build 11.0.8+11)
OpenJDK 64-Bit Server VM (build 11.0.8+11, mixed mode)
```
```
$ java -jar recaf-2.4.1-J8-jar-with-dependencies.jar
```
**Exception**
```
13:46:04.488 [main] INFO : Missing JavaFX dependencies, attempting to patch in missing classes
13:46:04.490 [main] INFO :  - Local cache found!
13:46:04.491 [main] INFO :  - Loading dependencies...
13:46:04.509 [main] INFO :  - Done!
13:46:04.509 [main] INFO : Recaf-2.4.1
2020-08-14 13:46:05.654 java[24770:368934] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
2020-08-14 13:46:05.654 java[24770:368934] CoreText note: Set a breakpoint on CTFontLogSystemFontNameRequest to debug.
2020-08-14 13:46:05.660 java[24770:368934] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
13:46:05.745 [main] INFO : Loaded workspace from: ReaderSDK-impl-en_GB.jar
2020-08-14 13:46:05.939 java[24770:368934] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
2020-08-14 13:46:06.013 java[24770:368984] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
...etc...
```

**Screenshots**
![image](https://user-images.githubusercontent.com/302127/90251520-c8b29080-de35-11ea-856d-becf728f5eee.png)

**Comment by tomtastic** [comment]
Possibly related bug : https://bugs.openjdk.java.net/browse/JDK-8234916

**Comment by Col-E** [comment]
I see the bug report for openjdk specifies Jfx 14... 

In the Recaf config directory, try replacing the JavaFx jars with updated ones:

https://repo1.maven.org/maven2/org/openjfx/javafx-controls/14.0.2.1/
https://repo1.maven.org/maven2/org/openjfx/javafx-graphics/14.0.2.1/
https://repo1.maven.org/maven2/org/openjfx/javafx-base/14.0.2.1/

If that fixes the problem the dependency pulling method just needs to pull higher versions.

**Comment by tomtastic** [comment]
Perfect!

![image](https://user-images.githubusercontent.com/302127/90294537-99744180-de7e-11ea-975b-056215b989fa.png)

**Comment by Col-E** [comment]
Fixed by @xxDark in the linked PR, which is available in [2.4.2](https://github.com/Col-E/Recaf/releases/tag/2.4.2)
